thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/MergedAnomalyResultBean.java

```diff
@@ -23,6 +23,7 @@
   private double weight; // change percentage, whose absolute value is severity
   private double impactToGlobal; // the impact of this anomaly to the global metric
   // Additional anomaly detection properties (e.g., patter=UP, etc.)
+  // Being used as identifying if two merged anomalies have same anomaly detection properties, thus can be mergeable
```

Comment:
Not all values in properties are used for determining isMergeable. The values to be used depends on the anomaly function. Please see my comment for the method, isMergeable, and redesign this part.